__tests__/components/editor/RDFModal.test.js

```diff
@@ -42,8 +44,8 @@ describe('<RDFModal />', () => {
     it('has a Modal.Footer', () => {
       expect(wrapper.find(Modal.Footer).length).toBe(1)
     })
-    it('has two Buttons (Cancel and Save)', () => {
-      expect(wrapper.find(Modal.Footer).find(Button).length).toBe(1)
```

Comment:
This is my favorite "whoops" -- the description says "two Buttons" and the actual test code expects 1.  I changed the test to something more useful, I think.